x-pack/plugins/license_management/public/index.ts

```diff
@@ -5,6 +5,7 @@
  */
 import { PluginInitializerContext } from 'src/core/public';
 
+export { LicenseManagementUIPluginSetup, LicenseManagementUIPluginStart } from './plugin';
```

Comment:
Do you mind putting the `exports` below the `imports` ?

Reply:
Changes in 73e832b65c118e802e2e5bb1a85d834bee95f5af